src/components/call-to-action-popup/call-to-action-popup.tsx

```diff
@@ -19,7 +19,12 @@ interface CallToActionPopupProps {
 const CallToActionPopup = withText({closeLabel: 'overlay.close'})(
   ({title, description, buttons, onClick, onClose, closeLabel}: CallToActionPopupProps) => {
     return (
-      <div className={styles.callToActionPopup} data-testid="call-to-action-popup">
+      <div
+        className={styles.callToActionPopup}
```

Comment:
did you test this with NVDA or JAWS ?
what text is read by the screen reader for the dialog ?
I'm asking because when we added a role to the cta overlay we had problems where the screen reader would announce only some of the texts, or announced the same text multiple times

Reply:
I've tested with both. In JAWS it reads correctly, but in the NVDA it reads the buttons twice. I'm wondering if it's because of the later focusing on the buttons

Reply:
For role dialog it announces only the buttons

Reply:
as Moshe told in the past, it's not matter if nvda read twice:

> If one screen reader - reads twice, then it is good enough as long as the info is not missing to the user, JAWS is our Anchor - it must work as expected on JAWS. (if NVDA reads twice or MAC-Voice over reads 3 times - it is good enough!)
Please follow these guidelines before deploying to QA.